app/controllers/hyrax/admin/collection_types_controller.rb

```diff
@@ -4,13 +4,31 @@ class Admin::CollectionTypesController < ApplicationController
       authorize! :manage, :collection_types
     end
 
+    layout 'dashboard'
+    class_attribute :form_class
+    self.form_class = Hyrax::Forms::Admin::CollectionTypeForm
+
     def index; end
 
-    def new; end
+    def new
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.configuration'), '#'
+      add_breadcrumb t(:'hyrax.admin.collection_types.index.breadcrumb'), hyrax.admin_collection_types_path
+      add_breadcrumb t(:'hyrax.admin.collection_types.new.header'), hyrax.new_admin_collection_type_path
+      @form = form_class.new
+    end
 
     def create; end
 
-    def edit; end
+    def edit
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+      add_breadcrumb t(:'hyrax.admin.sidebar.configuration'), '#'
+      add_breadcrumb t(:'hyrax.admin.collection_types.index.breadcrumb'), hyrax.admin_collection_types_path
+      add_breadcrumb t(:'hyrax.admin.collection_types.edit.header'), hyrax.edit_admin_collection_type_path
```

Comment:
I believe the `edit_admin_collection_type_path` method requires a `CollectionType` as a parameter.